common/postgresql-ng/templates/secrets.yaml

```diff
@@ -16,20 +16,26 @@ spec:
       - name: channel-config
         # at the time of writing, kubectl is only available in edge
         image: "{{ $.Values.global.dockerHubMirror }}/library/alpine:edge"
+        {{- $users := "" }}
+        {{- range $user, $settings := .Values.users }}
+        {{- $users = printf "%s postgres-user-password-%s" $users $user}}
+        {{- end }}
         command:
           - ash
           - -c
           - |
             set -eou pipefail
             apk add --no-cache --no-progress kubectl
 
-            # if we already have a secret, we can stop here
-            if [[ "$(kubectl get secrets postgres-password --ignore-not-found)" != "" ]]; then
-              exit 0
-            fi
+            for password in postgres-password {{ $users }}; do
+              # if we already have a secret, we can stop here
+              if [[ "$(kubectl get secrets "$password" --ignore-not-found)" != "" ]]; then
+                exit 0
+              fi
 
-            head -c 30 </dev/urandom | base64 > postgres-password
-            kubectl create secret generic postgres-password --from-file=postgres-password
+              head -c 30 </dev/urandom | base64 > "$password"
+              kubectl create secret generic "$password" --from-file="$password"
```

Comment:
This is not something that needs to be addressed in this PR, but we should think about setting `.metadata.ownerReferences` on these secrets to ensure that they are cleaned up when the Helm release is uninstalled (and also to ensure proper owner-info attribution).

Reply:
I added a todo for that